### ISSUE TITLE
unsetting $_FILES superglobal

### ISSUE THREAD

**Issue by i-just** [aelvan/Preparse-Field-Craft#52]
I have made a note of this issue on the closed ticekt as well: https://github.com/aelvan/Preparse-Field-Craft/issues/23 but figured I better open a new issue as well.

In Craft 3 a fix for the front-end asset uploads (more specifically the `resetUloads()` method in the `src/PreparseField.php` file) is causing the `symfony/http-foundation/Request::createFromGlobals()` method to fail as it cannot find `$_FILES` superglobal. In case of my website that causes serious problems when trying to make a payment.

Would changing `unset($_FILES)` to `$_FILES = []` still provide a fix for this issue? Second option doesn't break the `createFromGlobals()` or any other functionality that relies on the `$_FILES` superglobal being there.

**Comment by i-just** [comment]
Hi there,

Any chance you had time to review this issue?

Thanks,
Iwona

**Comment by aelvan** [comment]
Hi,

Not sure if that change would work or not. Setting up a test for it is a bit of work, so I haven't had time to look at it. I didn't experience the original issue, or wrote the fix, so if anyone wants to set up a test and confirm whether or not it works, that would be much appreciated.

**Comment by stenvdb** [comment]
I'm experiencing the same issue.

I tried to use this on a Commerce Order publish layout, because I wanted to include a specific customer primary address field in the Commerce export. My code for the preparse field I was using was:
`element.customer.primaryBillingAddress.businessTaxId|default('')`

But that gave me an `Undefined variable: _FILES` error when I tried to submit the order during the checkout. Am I doing something wrong?
